diff --git a/node_modules/@typescript-eslint/typescript-estree/dist/parseSettings/warnAboutTSVersion.js b/node_modules/@typescript-eslint/typescript-estree/dist/parseSettings/warnAboutTSVersion.js
index 3314fd7..3779959 100644
--- a/node_modules/@typescript-eslint/typescript-estree/dist/parseSettings/warnAboutTSVersion.js
+++ b/node_modules/@typescript-eslint/typescript-estree/dist/parseSettings/warnAboutTSVersion.js
@@ -80,12 +80,6 @@ function handleUnsupportedTSVersion(parseSettings, behavior, passedLoggerFn) {
     if (warnedAboutTSVersion) {
         return;
     }
-    if (passedLoggerFn ||
-        // See https://github.com/typescript-eslint/typescript-eslint/issues/7896
-        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-        (typeof process === 'undefined' ? false : process.stdout?.isTTY)) {
-        parseSettings.log(buildUnsupportedTSVersionMessage('warn'));
-    }
     warnedAboutTSVersion = true;
 }
 //# sourceMappingURL=warnAboutTSVersion.js.map
\ No newline at end of file
